ci-operator/step-registry/openshift/upgrade/qe/test/eus/openshift-upgrade-qe-test-eus-chain.yaml

```diff
@@ -1,20 +1,23 @@
 chain:
-  as: openshift-upgrade-qe-test-paused 
+  as: openshift-upgrade-qe-test-eus
   steps:
   - ref: idp-htpasswd
   - ref: cucushift-pre
   - ref: openshift-extended-upgrade-pre
   - ref: cucushift-upgrade-prepare
   - ref: cucushift-upgrade-prehealthcheck
-  - ref: cucushift-upgrade-paused-intermediate
-  - ref: cucushift-upgrade-paused-intermediate-sanity
-  - ref: cucushift-upgrade-paused-latest
-  - ref: cucushift-upgrade-paused-latest-sanity
-  - ref: cucushift-upgrade-paused-unpause
+  - ref: cucushift-upgrade-eus-pause-worker-mcp
+  - ref: cucushift-chainupgrade-toimage
+  - ref: cucushift-upgrade-toimage
+  - ref: cucushift-upgrade-eus-latest-sanity
+  - ref: cucushift-upgrade-eus-unpause-worker-mcp
   - ref: cucushift-upgrade-healthcheck
   - ref: openshift-extended-upgrade-post
   - ref: cucushift-upgrade-check
   - ref: openshift-upgrade-qe-test-report
+  env:
+  - name: E2E_RUN_TAGS
+    default: "@upgrade-sanity"
```

Comment:
Does it mean it will run e2e cases with tag `upgrade-sanity` in eus test?

Reply:
Right, beside that, in the [chainupgrade-toimage](https://github.com/openshift/release/pull/49959/files#diff-5f8db9edddddffd6ac79e9ee878d9d540bb1d3d623e27eaeff90bb97d95c26d9R637), it will append some other cluster related infrastructure tags.